dev-manual/env/compose.rst

```diff
@@ -195,6 +203,112 @@ stop and remove related containers, network and volumes by running:
+update the existing ``docker-compose.dev.yml`` file. These files are required to
+create a network for LDAP and to configure users for LDAP.
+
+First, create a custom network for authentication in a new file called
+``docker-compose.auth-network.yml`` and put it in the ``docker`` directory. Copy the
+following into the file:
+
+.. code-block:: bash
+
+   networks:
+      default:
+         name: auth-network
+         external: true
+
+Then set up docker compose to use the new network by appending the following to
+the end of the existing ``docker-compose.dev.yml`` file:
+
+.. code-block:: bash
+
+   networks:
+      auth-network:
+         name: auth-network
+
+Lastly, configure LDAP credentials in a LDAP Data Interchange Format (LDIF) file,
+let's call it ``config-ldap.ldif``. The following example creates 1 user, ``example``, with
+Administrator permissions. Please refer to the
+`LDAP documentation <https://ldap.com/ldif-the-ldap-data-interchange-format/>`__ for more help.
+
+.. code-block:: bash
+
+   dn: cn=example,ou=People,dc=example,dc=org
+   objectClass: person
+   objectClass: inetOrgPerson
+   sn: example
+   cn: example
+   mail: example@example.com
+   userpassword: example
+
+   dn: ou=Groups,dc=example,dc=org
+   objectClass: organizationalUnit
+   ou: Groups
+
+   dn: cn=Administrator,ou=Groups,dc=example,dc=org
+   objectClass: groupOfNames
+   cn: Administrator
+   member: cn=example,ou=People,dc=example,dc=org
+
+Finally, we can start AtoM using ``auth-network``:
+
+``docker compose -f docker-compose.yml -f docker/docker-compose.auth-network.yml up -d``
+
+AtoM Configuration
+++++++++++++++++++
+
+.. SEEALSO::
+
+   * :ref:`ldap-enabling`
+
+To enable LDAP Authentication, change the ``myUser`` value to ``ldapUser`` in
+``config/factories.yml``:
+
+.. code-block:: bash
+
+    user:
+      class: ldapUser
+
+Clear cache and restart ``atom_worker`` for the changes to appear. After doing so,
+a new configuration section will be available in |gears| **Admin > Settings**,
+where you can define your LDAP authentication settings:
+
+.. image:: images/ldap-config.*
+   :align: center
+   :width: 90%
+   :alt: An image of the LDAP authentication settings
+
+To determine the Host IP for this example based on your docker configuration, run
+``docker inspect auth-network`` to determine the Host IP (``IPv4Address``) used
+for ``openldap``.
+
+.. NOTE::
+
+   The Host IP value is dynamic and may be different each time.
+
+Other values that we will use for this example are as follows:
+
+    - Port: 389
+    - Base DN: ou=People,dc=example,dc=org
+    - Bind Lookup Attribute: cn
+
+.. NOTE::
+
+   Although we've defined the openldap port to ``1389`` in ``docker-compose.dev.yml``,
+   ``LDAP_PORT`` should still be running on port ``389``. This can be confirmed
+   by running ``docker compose logs openldap``, and you should be able to see the
+   defined ``LDAP_PORT``.
+
```

Comment:
This could be part of a README to start up a Dockerized OpenLDAP test instance. I don't think this needs to be part of the AtoM docs.